task_3.py

```diff
@@ -0,0 +1,14 @@
+# Создайте функцию генератор чисел Фибоначчи (см. Википедию)
+
+a = int(input('Напишите число: '))
+
+def fib_gen():
+    a, b = 0, 1
+    while 1:
```

Comment:
Принятые использовать `True`. Вместо любого другого значения, болевое представление которого также `True`.
```suggestion
    while True:
```